cli/cmd/upgrade_project.go

```diff
@@ -141,19 +144,25 @@ For more information about upgrading projects, go to [Manage Keptn](https://kept
 			return err
 		}
 
-		shipyardName := "shipyard.yaml"
-		upgradedShipyardResource := &apimodels.Resource{
-			ResourceContent: string(marshalledUpgradedShipyard),
-			ResourceURI:     &shipyardName,
+		existingProject, getPrjErr := projectsHandler.GetProject(apimodels.Project{ProjectName: projectName})
+		if getPrjErr != nil {
+			return errors.New(*getPrjErr.Message)
 		}
-		if _, err := resourceHandler.UpdateProjectResource(projectName, upgradedShipyardResource); err != nil {
-			return fmt.Errorf("could not update shipyard resource: %s", err.Error())
+
+		UpdateProject := apimodels.CreateProject{
+			GitRemoteURL: existingProject.GitRemoteURI,
+			GitToken:     existingProject.GitToken,
```

Comment:
the GitToken of the existing project retrieved from the API will always be empty, since this property is not stored in the database. So there might be the danger of deleting existing credentials when the project is updated. Is this case handled in the project API?